spec/lib/chess/core/cell_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+RSpec.describe Chess::Core::Cell do
+  let(:free_cell) { described_class.new('a1', :white) }
+  let(:occupied_cell) { described_class.new('b2', :white) }
+  let(:piece) do
+    Chess::Core::Pieces::BasePiece.create_and_occupy(%i[R N B Q K P], [Chess::WHITE_TEAM, Chess::BLACK_TEAM].sample,
+                                                     occupied_cell)
```

Comment:
```suggestion
    Chess::Core::Pieces::BasePiece.create_and_occupy(
      %i[R N B Q K P].sample,
      [Chess::WHITE_TEAM, Chess::BLACK_TEAM].sample,
      occupied_cell  
    )
```

Reply:
Yes, and let us use .new to instantiate the BasePiece because this doesn't occupy the cell. And we can do better tests on cell